streaming/java/dependencies.bzl

```diff
@@ -14,6 +14,10 @@ def gen_streaming_java_deps():
             "org.slf4j:slf4j-log4j12:1.7.25",
             "org.apache.logging.log4j:log4j-core:2.8.2",
             "org.testng:testng:6.9.10",
+            "log4j:log4j:1.2.17",
+            "org.mockito:mockito_all:1.10.19",
+            "org.apache.commons:commons-lang3:3.3.2",
+            "com.twmacinta:fast-md5:2.7.1",
```

Comment:
Not found on repo1.maven.org/maven2

```
not found: https://repo1.maven.org/maven2/com/twmacinta/fast-md5/2.7.1/fast-md5-2.7.1.pom
```

it seems not a typo but the artifact missing.

Reply:
Possibly we use other MD5 utility like commons-codec

```
"commons-codec:commons-codec:1.14"
```

Reply:
[patch.txt](https://github.com/ray-project/ray/files/4298463/patch.txt)

Here is a patch that you can use to make bazel settings work.

Reply:
I'll recheck the pom. If the file is deleted, I'll upgrade or find another way to calculate md5.